### ISSUE TITLE
Allow for projects=all when running the install generator in non-interactive mode

### ISSUE THREAD

**Issue by jdgeier** [jscutlery/semver#503]
### Discussed in https://github.com/jscutlery/semver/discussions/491

<div type='discussions-op-text'>

<sup>Originally posted by **jdgeier** April 25, 2022</sup>
Hello, I know that when I run the install generator it basically sets everything to all by default but I have to run the generator in interactive mode in order for that to work correctly. I'd like to run the command prior to tagging an affected:version to insure all packages in the project get built, tagged, and deployed. Basically I would like to run the following:

`npx nx g @jscutlery/semver:install --interactive=false --enforce-conventional-commits --projects=all`</div>

**Comment by edbzn** [comment]
@jdgeier btw, thanks a lot for the sponsoring. :100:

**Comment by edbzn** [comment]
Yeah I think it's normal as the root is defined in workspace.json with standalone config.

**Comment by jdgeier** [comment]
> Yeah I think it's normal as the root is defined in workspace.json with standalone config.

It seems to be breaking the affected command on older versions of nx. Do you know when it started removing root from the project.json?
